support-frontend/assets/components/subscriptionCheckouts/layout.jsx

```diff
@@ -8,21 +8,37 @@ import LeftMarginSection from 'components/leftMarginSection/leftMarginSection';
 type PropTypes = {
   children: Node,
   aside: ?Node,
+  footerAside: ?Node,
 };
-const CheckoutLayout = ({ children, aside }: PropTypes) => (
-  <div className={[styles.root, aside ? styles.withAside : null].join(' ')}>
-    {aside &&
+const CheckoutLayout = ({ children, aside, footerAside }: PropTypes) => {
+  let mainClass = styles.root;
+  if (aside) {
+    mainClass = [styles.root, styles.withAside].join(' ');
+  } else if (footerAside) {
+    mainClass = [styles.root, styles.withFooterAside].join(' ');
+  }
```

Comment:
The slight awkwardness of this bit of code makes me wonder whether this would be easier to refactor this component using emotion - the styles could just be passed into the sections without all the ifs and buts.

Reply:
I think this can be improved a lot, I'll take a look
I'm always slightly wary of moving to emotion for things which are widely used because it's so hard to test that you haven't accidentally changed something

Reply:
Yeah, I think there's an argument for keeping this as it is as the code could get spread out and hard to maintain.

Reply:
I think this is also where we would be better served by object styles, as they lend themselves better to being combined.